packages/create-sitecore-jss/src/templates/react/package.json

```
@@ -114,5 +114,10 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "overrides": {

```

You can add a link to existing cra issue so it will explain the reason of these overrides:
```suggestion
  // Resolves security vulnerabilities: https://github.com/facebook/create-react-app/issues/17120
  "overrides": {

```

---

@faiqamin I added a draft example, you can't write comments in json file using this approach